### PR TITLE
PR for issue Dap-19

### PR DESCRIPTION
**Pull request by JohanBester**
OK folks,
This PR is ready for review.
Kindly please look at these changes I made to section 5 and associated files.
Thank you all for your collaboration.
Johan

### CODE REVIEW

**Review by coyoteJohn — DISMISSED** [review]
Looks good.

File: Section05/5.1-JupyterNotebooks_PythonIntro.ipynb [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+    "\n",
+    "#### Well, what is a Programer?\n",
+    "\n",
+    "A computer programmer or a coder, is a person who creates computer programs — often to solve large problems.\n",
+    "\n",
+    "As such, a programer is a creator, a problem solver, and a thinker.\n",
+    "\n",
+    "#### And what is an Data Analyst?\n",
+    "\n",
+    "A Data Analysts conducts research and analyze collected data to find patterns, trends, and insights that can help answer question in decision-making and planning.\n",
```
Comment (Danielle0727) [inline Section05/5.1-JupyterNotebooks_PythonIntro.ipynb:42]: Suggested edit "A Data Analyst conducts research and analyzes collected data....can help answer questions" I removed the s from Analyst and added an s to analyze and question

File: Section05/5.1-JupyterNotebooks_PythonIntro.ipynb [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+    "\n",
+    "## Jupyter Notebook\n",
+    "\n",
+    "### What is Jupyter Notebook?\n",
+    "\n",
+    "Jupyter Notebook is an open source web application used to create and share documents that contain ...\n",
+    "- live code \n",
+    "- equations \n",
+    "- visualizations \n",
+    "- text.\n",
```
Comment (Danielle0727) [inline Section05/5.1-JupyterNotebooks_PythonIntro.ipynb:304]: Noticed a period after "text" only.

File: Section05/5.2-Python_DataTypes.ipynb [hunk 3]
```diff
@@ -192,16 +193,20 @@
     "\n",
     "## Python Data Types\n",
     "\n",
-    "| Example | Data Type |\n",
+    "| Examples of data types | Data Type |\n",
     "|---------|-----------|\n",
     "| `x = “Hello World”` | string (str) |\n",
     "| `x = 20` | integer (int) |\n",
-    "| `x = 20.5` | float |\n",
+    "| `x = 20.5` | Floating-Point Numbers (float) |\n",
```
Comment (Danielle0727) [inline Section05/5.2-Python_DataTypes.ipynb:200]: Might be helpful for the students to state that floating numbers mean numbers with a decimal point.

File: Section05/5.1-JupyterNotebooks_PythonIntro.ipynb [hunk 4]
```diff
@@ -0,0 +1,481 @@
...
+    "The languages that are most easily understood by computers are the \"lowest level languages\" known as **Machine Languages**. These Machine Languages decode instructions into binary code that is easily interpreted and executed by computer hardware.\n",
+    "\n",
+    "While computers love low-level / machine languages, for us humans, it's easier to use a high-level coding language.\n",
+    "\n",
+    "High-level coding languages are more like natural languages\n",
+    "-   they take less time to write\n",
+    "-   they are shorter and easier to read\n",
+    "-   and they are more likely to be correct/error free when we run them\n",
+    "\n",
+    "The drawback with High-level languages is that they need to be “translate” into machine language for computers to understand.\n",
```
[the remark this answers is not in the document]
Comment (JohanBester) [reply Section05/5.1-JupyterNotebooks_PythonIntro.ipynb:82]: Changed it to translated

File: Section06/6.1-Python_FlowControl.ipynb [hunk 5]
```diff
@@ -48,34 +51,56 @@
...
+   "source": [
     "`If / Else` statements are used along with the Logical Comparison Operators we looked at earlier.\n",
     "\n",
-    "### **Comparison Operators**\n",
+    "**Comparison Operators**\n",
     "\n",
     "| Operator | Name | Example |\n",
     "|----|----|----|\n",
-    "| == | Equal | a == b |\n",
+    "| == | **Equal** | a == b |\n",
```
[the remark this answers is not in the document]
Comment (JohanBester) [reply Section06/6.1-Python_FlowControl.ipynb]: Removed the bold highlight

File: Section05/5.1-JupyterNotebooks_PythonIntro.ipynb (hunk 2 above)
Comment (JohanBester) [reply Section05/5.1-JupyterNotebooks_PythonIntro.ipynb:304]: Removed the period

File: Section05/5.2-Python_DataTypes.ipynb (hunk 3 above)
Comment (JohanBester) [reply Section05/5.2-Python_DataTypes.ipynb:200]: Added that comment about decimal point

File: Section05/5.1-JupyterNotebooks_PythonIntro.ipynb (hunk 1 above)
Comment (JohanBester) [reply Section05/5.1-JupyterNotebooks_PythonIntro.ipynb:42]: Thank you, I fixed the grammar and spelling

**Review by matsinet — APPROVED** [review]
Ship it

**Pull request merged**